packages/renderer/src/lib/ingresses-routes/IngressRouteColumnBackend.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+import type { IngressUI } from './IngressUI';
+import type { RouteUI } from './RouteUI';
+
+export let object: IngressUI | RouteUI;
+</script>
+
+{#if 'rules' in object && object.rules}
+  {#each object.rules as rule}
```

Comment:
Maybe move the logic to ingress-route-utils.ts:getRouteUI()/getIngressUI() and stored as string[] in IngressUI/RouteUI element? Would keep the Kube logic in one place and this component simple.